### PR TITLE
[WIP] Update version table and testing matrices

### PR DESCRIPTION
**Pull request by atombrella**
`travis.yml` has Django 2.2 in it, so I guess it's left out by mistake?

### CODE REVIEW

**Comment by benjaoming** [comment]
Thanks @atombrella 

Release 0.4.x doesn't support 2.2 officially. Current master targets 0.5.

See: https://github.com/django-wiki/django-wiki/blob/releases/0.4.x/.travis.yml

**Comment by atombrella** [comment]
@benjaoming OK. Can I add the `0.5 (master)` line to the table?

File: tox.ini [hunk 1]
```diff
@@ -34,16 +31,13 @@ usedevelop = false
 
 deps =
   .[test]
-  django111: Django>=1.11,<2.0
-  django20: Django>=2.0,<2.1
-  django21: Django>=2.1,<2.2
   django22: Django>=2.2,<2.3
+  django20: Django>=3.0a1,<3.1
```
Comment (benjaoming) [inline tox.ini]: Should say `django30`

**Comment by benjaoming** [comment]
Maybe supporting 2.1 is okay? It's supported until end of 2019 and may provide a nice overlap for upgrading (so you don't have to upgrade Django AND django-wiki at the same time).

File: tox.ini (hunk 1 above)
Comment (atombrella) [reply tox.ini]: Yes, I noticed myself :) I'll push some updates today, and we'll see how it works. It seems that no additional work needs to be done to support Django 3.0. I might be wrong, though.

**Comment by atombrella** [comment]
> Maybe supporting 2.1 is okay? It's supported until end of 2019 and may provide a nice overlap for upgrading (so you don't have to upgrade Django AND django-wiki at the same time).

Perhaps. What's the timeline for Django Wiki 0.5? What do you think about Python 3 only for Django Wiki 0.5?

**Comment by atombrella** [comment]
1.11 is LTS, 2.2 is LTS, so there's probably a good chance that only having official support for the two newest Django releases works.

**Comment by benjaoming** [comment]
> Perhaps. What's the timeline for Django Wiki 0.5?

I haven't recorded anything grave lately, so I guess shipping it after this change is a good possibility!

You have any opinions or inputs on that decision?

**Comment by atombrella** [comment]
No. Shipping soon would be good. Then Django 2.1 makes sense.

**Pull request merged**